src/hooks/tapDoneToAsyncGetIssues.ts

```diff
@@ -0,0 +1,70 @@
+import webpack, { compilation } from 'webpack';
+import { ForkTsCheckerWebpackPluginConfiguration } from '../ForkTsCheckerWebpackPluginConfiguration';
+import { ForkTsCheckerWebpackPluginState } from '../ForkTsCheckerWebpackPluginState';
+import { getForkTsCheckerWebpackPluginHooks } from './pluginHooks';
+import { OperationCancelledError } from '../reporter';
+import { createWebpackFormatter } from '../formatter/WebpackFormatter';
+import { Issue } from '../issue';
+import isPending from '../utils/async/isPending';
+import wait from '../utils/async/wait';
+import chalk from 'chalk';
+
+function tapDoneToAsyncGetIssues(
+  compiler: webpack.Compiler,
+  configuration: ForkTsCheckerWebpackPluginConfiguration,
+  state: ForkTsCheckerWebpackPluginState
+) {
+  const hooks = getForkTsCheckerWebpackPluginHooks(compiler);
+
+  compiler.hooks.done.tap('ForkTsCheckerWebpackPlugin', async (stats) => {
+    const report = state.report;
+    let issues: Issue[];
+
+    try {
+      if (await isPending(report)) {
+        hooks.waiting.call(stats.compilation);
+        configuration.logger.issues.log(chalk.blue('Issues checking in progress...'));
+      } else {
+        // wait 10ms to log issues after webpack stats
+        await wait(10);
+      }
+
+      issues = await report;
+    } catch (error) {
+      if (!(error instanceof OperationCancelledError)) {
+        hooks.error.call(error, stats.compilation);
```

Comment:
Minor thing, I wonder if this might be easier to read if the cases were reversed:

```suggestion
	      if (error instanceof OperationCancelledError) {
```

For my money that's easier to understand than `(!(error...`

Just a thought

Reply:
:+1: